signcrypt_seal.go

```diff
@@ -69,6 +69,7 @@ func (sss *signcryptSealStream) signcryptBytes(b []byte) error {
 
 	nonce := nonceForChunkSigncryption(sss.numBlocks)
 
+	// TODO: Encode payload chunk.
```

Comment:
I'm looking at the saltpack 2.0 spec, which says:

> Signcryption payload packets won't look as much like their v1 DH counterparts, because there aren't any per-recipient authenticators. Here's the packet format:
> [
>   abc123...,  // signcrypted payload chunk
> ]
> For consistency with the other modes, and for extensibility just in case, payload packets are a messagepack list. However the list contains a single binary chunk.